src/test/java/StringTest.java

```diff
@@ -0,0 +1,44 @@
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class StringTest {
+
+    @DisplayName("요구사항1 - split 테스트")
+    @Test
+    void testSplit() {
+        String value = "1,2";
+        String[] split = value.split(",");
+        assertThat(split).hasSize(2);
+        assertThat(split).contains("1", "2");
+        assertThat(split).containsExactly("1", "2");
+
+        String value2 = "1";
+        String[] split2 = value2.split(",");
+        assertThat(split2).hasSize(1);
+        assertThat(split2).contains("1");
+        assertThat(split).containsExactly("1");
+    }
+
+    @DisplayName("요구사항2 - substring 테스트")
+    @Test
+    void testSubstring() {
+        String value = "(1,2)";
+        String substring = value.substring(1, value.length() - 1);
```

Comment:
이 경우에 변수는 inline 으로 적용되면 가독성이 더 좋을 것 같습니다.
```suggestion
        String substring = "(1,2)".substring(1, value.length() - 1);
```